pom.xml

```diff
@@ -275,6 +276,12 @@
                 <version>${powermock.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
@kezhenxu94 Should I move the `lombok` into root `dependencies` other than `dependencyManagement`.

Reply:
That depends on how you use it. Logically, they are the same. But `provided` is pointless.

Reply:
And from the current `dependencies` in the root `pom.xml`, no lib is declared there.

Reply:
It should be better, and remove all Lombok in all sub modules